### DISCUSSION TITLE
Gsoc'23- Related to Auto-detect coverage bounding boxes for brain MRI images

### DISCUSSION THREAD

**Discussion by Shazam213** [Emory-HITI/Niffler#379] [General]
Hello all, as mentioned in the project description i downloaded one of the publicly available MRI data from [the Cancer Imaging Archive (TCIA)](https://www.cancerimagingarchive.net/)  and converted the DICOM data to png using the png-extraction module of Niffler. Then applied some basic OpenCV functions on the png data like drawing the bounding boxes manually as part of initial testing as suggested by @pradeeban (reference- https://github.com/NISYSLAB/Emory-BMI-GSoC/discussions/5#discussioncomment-4871145).

Could you kindly advise me on the next steps to take or any tasks that need to be completed?

**Comment by pradeeban** [comment]
You have completed the pre-requisites successfully. Please make sure to demonstrate these in the proposal. (Not just here, since we won't remember your message at the time of ranking the proposals).

It is important to elaborate the steps you have followed in the proposal as well, to highlight your understanding of the problem (and the pre-requisite that you have completed).

**Comment by Shazam213** [comment]
@pradeeban Hello sir, I have finished preparing my proposal for the project Auto-detect coverage bounding boxes for brain MRI images. Could you kindly review it before submitting it and let me know if any changes are required? If so, could you please inform me how I can share it with you?

**Reply by pradeeban** [reply]
My and other mentors' email addresses are included under each project idea. Please email them as a Google Doc link (with comments enabled).

**Reply by Shazam213** [reply]
thank you sir
